plugins/reach/lib/kReachManager.php

```diff
@@ -34,6 +34,7 @@ protected function getObjectType($eventObjectClassName)
 			"categoryEntry" => objectType::CATEGORY_ENTRY,
 			"CaptionAsset" => CaptionAssetEventNotificationsPlugin::getEventNotificationEventObjectTypeCoreValue(CaptionAssetEventNotificationEventObjectType::CAPTION_ASSET),
 			"TranscriptAsset" => TranscriptAssetEventNotificationsPlugin::getEventNotificationEventObjectTypeCoreValue(TranscriptAssetEventNotificationEventObjectType::TRANSCRIPT_ASSET),
+			"LiveStreamScheduleEvent" => ScheduleEventNotificationsPlugin::getEventNotificationEventObjectTypeCoreValue(ScheduleEventNotificationEventObjectType::SCHEDULE_EVENT),
```

Comment:
LiveStreamScheduleEvent -> ScheduleEvent (as that the type you take the enum from)

Reply:
@david-winder-kaltura  I've tested this. The objectType that the flow get is LiveStreamScheduleEvent and if I put the generic calssname here, the flow will ignore the event

Reply:
I think it can be confuse but because this map also contain LiveStreamEntry -> entry I guess we can live with that as well